bigquery_etl/cli/query.py

```diff
@@ -1530,49 +1530,53 @@ def _initialize(query_file):
     type=click.Path(file_okay=False),
     required=False,
 )
-def render(name, sql_dir, output_dir):
+@parallelism_option()
+def render(name, sql_dir, output_dir, parallelism):
     """Render a query Jinja template."""
     if name is None:
         name = "*.*"
 
     query_files = paths_matching_name_pattern(name, sql_dir, project_id=None)
     resolved_sql_dir = Path(sql_dir).resolve()
-    for query_file in query_files:
-        table_name = query_file.parent.name
-        dataset_id = query_file.parent.parent.name
-        project_id = query_file.parent.parent.parent.name
-
-        jinja_params = {
-            **{
-                "project_id": project_id,
-                "dataset_id": dataset_id,
-                "table_name": table_name,
-            },
-        }
 
-        rendered_sql = (
-            render_template(
-                query_file.name,
-                template_folder=query_file.parent,
-                templates_dir="",
-                format=False,
-                **jinja_params,
-            )
-            + "\n"
+    with Pool(parallelism) as p:
+        p.map(partial(_render_query, output_dir, resolved_sql_dir), query_files)
+
+
+def _render_query(output_dir, resolved_sql_dir, query_file):
+    table_name = query_file.parent.name
+    dataset_id = query_file.parent.parent.name
+    project_id = query_file.parent.parent.parent.name
+
+    jinja_params = {
+        **{
+            "project_id": project_id,
+            "dataset_id": dataset_id,
+            "table_name": table_name,
+        },
+    }
```

Comment:
This could be simplified.
```suggestion
    jinja_params = {
        "project_id": project_id,
        "dataset_id": dataset_id,
        "table_name": table_name,
    }
```